Topics/starting-a-data-project.md

```diff
@@ -0,0 +1,254 @@
+# Starting a Data Project
+
+# What you will learn 
+In this guide, you'll learn to prepare yourself to start the project. 
+
+This will consist of learning how to look for sources that can help you solve the problem, building the project structure, build good documentation, secure your code, our data and more!
```

Comment:
> build good documentation, secure your code, our data and more!

building good documentation, securing your code and our data and others!  --- how about this?

Reply:
I don't understand, can you explain better?

Reply:
I was suggesting a new sentence as opposed to what you wrote.

Your original sentence:
> build good documentation, secure your code, our data and more!

My suggestion:
> building good documentation, securing your code and our data and others!

Does this above format look better for future suggestions? I thought I had used the previous one in the past and it was clear, let me know which one?

Reply:
got t now , I didn t notice the difference within the tenses :D

Reply:
it is better with this new format.

Reply:
It's not the new format it's using the tool incorrectly on your part!